src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -627,6 +627,197 @@ void Do<T>(out T goo)
             await VerifyItemExistsAsync(markup, "test");
         }
 
+
+        [WorkItem(22342, "https://github.com/dotnet/roslyn/issues/22342")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async void TupleExpressionDeclaration1()
```

Comment:
The test name is confusing. This is a tuple type, not a tuple expression.
I think the same applies for some other tests.

Reply:
use async Task for all tests

Reply:
BTW, the existing tests are already fixed if you merge latest master. Please remove async void from your tests too.

Reply:
You are right. The reason for this is that the code under test is parsed as a tuple expression.
![2018-05-25 11_55_57-roslyn](https://user-images.githubusercontent.com/24472128/40538726-be872e34-6012-11e8-8496-dc9566441eff.png)
I haven't figured out how to find better names for such the tests.

Reply:
@Neme12 Done (e752f3c). Thanks for also pointing out the already fixed tests in master.